openedx/openedx-platform
# feat: set collections for a library component [FC-0062]

## Description

Adds api to set/update collections on a given component. Related to https://github.com/openedx/frontend-app-authoring/pull/1373


## Supporting information

- `Private-ref`: https://tasks.opencraft.com/browse/FAL-3848
- https://github.com/openedx/frontend-app-authoring/pull/1373
- https://github.com/openedx/openedx-learning/pull/238

**Depends on:**

* https://github.com/openedx/openedx-events/pull/404

## Testing instructions

See https://github.com/openedx/frontend-app-authoring/pull/1373 for instructions.

## Deadline

"None" if there's no rush, or provide a specific date or event (and reason) if there is one.

## Other information
